### ISSUE TITLE
How to position tooltip on action items in the toolbar?

### ISSUE THREAD

**Issue by grennis** [spyhunter99/supertooltips#10]
Hi, we need to show tooltips to help user understand what the toolbar buttons do (in the top right corner of the screen). I'm not sure is it possible with this library? Library seems to require positioning relative to a view.

**Comment by spyhunter99** [comment]
Toolips are anchored to a view, such as EditText, TextView, Button, etc.
Those all extend view.

**Comment by grennis** [comment]
Yes I found I can get the toolbar and find the children in it and that works. Thanks!

**Comment by grennis** [comment]
Actually I found this is not the case. The action items in the toolbar are represented by one view (ActionMenuView) and therefore I can't position the tooltip on one item inside it. I think I will have to create a ghost placeholder view and manually position it and then position the tooltip relative to that. I think the library should support this for me, either directly supporting toolbar action items or at least allowing me to specify an X, Y instead of a view.
